app/plugin_api.go

```diff
@@ -279,6 +279,14 @@ func (api *PluginAPI) CopyFileInfos(userId string, fileIds []string) ([]string,
 	return api.app.CopyFileInfos(userId, fileIds)
 }
 
+func (api *PluginAPI) GetFileInfo(fileId string) (*model.FileInfo, *model.AppError) {
+	return api.app.GetFileInfo(fileId)
+}
+
+func (api *PluginAPI) ReadFileAtPath(path string) ([]byte, *model.AppError) {
```

Comment:
Wondering if it would be better to just name it `ReadFile`?

Reply:
Want me to do the change? I'm 0/5 really

Reply:
Yeah let's change it